editor/debugger/editor_visual_profiler.cpp

```diff
@@ -70,7 +70,7 @@ void EditorVisualProfiler::add_frame_metric(const Metric &p_metric) {
 	updating_frame = true;
 	clear_button->set_disabled(false);
 	cursor_metric_edit->set_max(frame_metrics[last_metric].frame_number);
-	cursor_metric_edit->set_min(MAX(frame_metrics[last_metric].frame_number - frame_metrics.size(), 0u));
+	cursor_metric_edit->set_min(MAX(int64_t(frame_metrics[last_metric].frame_number) - frame_metrics.size(), 0u));
```

Comment:
This doesn't seem to trigger a compiler warning, but technically the first argument to `MAX` is now `int64_t`, so it should probably not use the unsigned `0u` as second argument.
```suggestion
	cursor_metric_edit->set_min(MAX(int64_t(frame_metrics[last_metric].frame_number) - frame_metrics.size(), 0));
```

Reply:
It is `constexpr auto MAX(const T m_a, const T2 m_b)` so the arguments doesn't have to be the same type, and I assume that greater-than will work as expected between signed and unsigned. But this feels sloppy so I will push a correction.

There is no way I can just accept your suggestion and squash it right here in the Github UI, right?

Reply:
> There is no way I can just accept your suggestion and squash it right here in the Github UI, right?

Sadly no :/